src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -5,18 +5,23 @@ import React, {
   CSSProperties,
   useContext,
   useMemo,
+  useState,
 } from 'react';
 import { Vector3 } from 'three';
 import { ReactThreeFiber, PointerEvent, useThree } from 'react-three-fiber';
 import { clamp } from 'lodash-es';
+import { useComlink } from 'react-use-comlink';
 import { useMeasure } from 'react-use';
-import { scaleSymlog, scaleLinear, scaleSequential } from 'd3-scale';
-import { rgb } from 'd3-color';
 import shallow from 'zustand/shallow';
 import { useHeatmapConfig } from './config';
 import { HeatmapProps, HeatmapContext } from './HeatmapProvider';
-import { DataScale, D3Interpolator } from './models';
+import { D3Interpolator } from './models';
 import { INTERPOLATORS } from './interpolators';
+import { TextureWorker } from './worker';
+
+// eslint-disable-next-line
+// @ts-ignore
+import Worker from 'worker-loader!./worker'; // eslint-disable-line
```

Comment:
`eslint` problems ?

Reply:
🙄 and create-react-app, which doesn't include `worker-loader`...

Reply:
... ad Jest, which doesn't really like webpack's inline loader syntax...